src/components/UserContributionForm.vue

```diff
@@ -51,19 +51,22 @@ const submit = async () => {
           alert(res.data);
         })
         .catch((error) => {
-          if (error.response.status === 409) {
+          if (error.response.status === 404) {
             alert(
-              "Someone else has completed this task.  Please click 'skip to next' to try another one."
+              "The task cannot be found.  It may have already been completed.  Please click 'skip to next' to try another one."
             );
-          }
-          if (error.response.status === 401) {
+          } else if (error.response.status === 401) {
             alert(
               "You must be logged in to submit a task. Please click on the Login button."
             );
-          }
-          alert(
-            "There was a problem sending your request. Please try again later."
-          );
+          } else if (error.response.status === 400) {
```

Comment:
should we use a swich statement or better still an object for this response status options?

Reply:
Addressed with [1ee079a](https://github.com/wikimedia/toolhunt-ui/pull/57/commits/1ee079aae2f807905904859fb1b65ad8d1ed020a)

Wow, that is a lot cleaner!